editors/code/src/test_explorer/README.md

```diff
@@ -0,0 +1,67 @@
+### Trade-off
+There are many strategies about when to send what requests.
+
+Like the laziness is a big choice. When would you like to load how many tests?
+
+An obvious choice to to load all tests for all projects at the first time, then update the changed files.(this is what's used now)
```

Comment:
Yeah, laziness should be better choice.

We load all for now because it's simpler. After refactoring, it should be more easier to be laziness, I will finish it after basic features are all finished.

Reply:
We are lazily now